src/App.js

```diff
@@ -29,8 +29,25 @@ class App extends Component {
 
   showBucketList = (event) => {
     event.preventDefault();
+    const cachedBucketListKeys = localStorage.getItem('bucketList')
+    let bucketListParks;
+    if (cachedBucketListKeys) {
+      let bucketListKeys = JSON.parse(cachedBucketListKeys);
+      bucketListParks = this.state.parks.reduce((arr, park) => {
```

Comment:
I think we can use a filter and .includes() here on this.state.parks to DRY up the code instead of a reduce and forEach. 
`bucketListParks = this.state.parks.filter(park => bucketListKeys.includes(park.urlCode)`

Reply:
Thank you! Yeah agreed that makes this function a lot easier to read through too